lib/lita/adapters/slack/api.rb

```diff
@@ -47,6 +47,39 @@ def im_list
           call_api("im.list")
         end
 
+        def conversations_list(types: ["public_channel"], page_limit: nil)
+          result = call_api(
+            "conversations.list",
+            {
+              limit: page_limit,
+              types: types.join(",")
+            }
+          )
+
+          next_cursor = result.dig("response_metadata")&.dig("next_cursor")
+          next_cursor&.sub!("=", "%3D")
```

Comment:
Would it be more clear to future people why this is needed if you use something like `ERB::Util.url_encode` instead of string substitution?

Reply:
You’re right, I was going strictly to the API docs, I’ll switch it in The morning

Reply:
Fixed, did a refactor to DRY up some of the code too